spec/outputs/webhdfs_spec.rb

```diff
@@ -0,0 +1,142 @@
+# encoding: utf-8
```

Comment:
this test are integration test, this should be move into an specific folder, so something like ```spec/outputs/integration/``` you can take inspiration to what has been done for https://github.com/logstash-plugins/logstash-input-s3/tree/master/spec/integration